core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/visualization/hierarchychart/HierarchyChartOpDesc.scala

```diff
@@ -44,14 +43,19 @@ class TreeMapOpDesc extends VisualizationOperator with PythonOperatorDescriptor
   @JsonPropertyDescription("hierarchy of sectors, from root to leaves")
   var hierarchy: List[HierarchySection] = List()
 
+  @JsonProperty(required = true)
+  @JsonSchemaTitle("Chart Type")
+  @JsonPropertyDescription("treemap or sunburst")
```

Comment:
It is not a good idea to put the value column first because newly added columns will make the chart type invisible which is not user friendly. Also the description is not accurate to use "tree", it is not treemap only any more.

Reply:
Please review the recent changes:
1. The order of properties has been changed for user-friendliness (Order: Chart Type -> Hierarchy Path -> Value Column).
2. The description has been improved to be more comprehensive.